_partials/_cloud-data-tiering-beta.md

```diff
@@ -5,7 +5,7 @@ To request access to the private beta for data tiering:
     account](https://console.cloud.timescale.com/).
 1.  Click on the service where you want to enable data tiering.
 1.  Navigate to `Operations`, then `Data tiering`.
-1.  Click on `Request access`. An email goes to our support team, who can
+1.  Click on `Request access` to send an email to the support team, who can
```

Comment:
The flow needs change.

Reply:
Left a comment for this https://github.com/timescale/docs/pull/2163#issuecomment-1478065507

Reply:
This partial is no longer included on the data tiering pages.

Reply:
Yep the early access partial should work for this feature